data.sql

```diff
@@ -34,29 +34,153 @@ UPDATE animals SET owner_id=3 WHERE name IN ('Devimon','Plantmon');
+    '2020-03-08'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+    (SELECT id FROM animals WHERE name = 'Pikachu'),
+    '2020-05-14'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+    (SELECT id FROM animals WHERE name = 'Devimon'),
+    '2021-05-04'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'),
+    (SELECT id FROM animals WHERE name = 'Charmander'),
+    '2021-02-24'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+    (SELECT id FROM animals WHERE name = 'Plantmon'),
+    '2019-12-21'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'William Tatcher'),
+    (SELECT id FROM animals WHERE name = 'Plantmon'),
+    '2020-08-10'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+    (SELECT id FROM animals WHERE name = 'Plantmon'),
+    '2021-04-07'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+    (SELECT id FROM animals WHERE name = 'Squirtle'),
+    '2019-09-29'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'),
+    (SELECT id FROM animals WHERE name = 'Angemon'),
+    '2020-10-03'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'),
+    (SELECT id FROM animals WHERE name = 'Angemon'),
+    '2020-11-04'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+    (SELECT id FROM animals WHERE name = 'Boarmon'),
+    '2019-01-24'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+    (SELECT id FROM animals WHERE name = 'Boarmon'),
+    '2019-05-15'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+    (SELECT id FROM animals WHERE name = 'Boarmon'),
+    '2020-02-27'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+    (SELECT id FROM animals WHERE name = 'Boarmon'),
+    '2020-08-03'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+    (SELECT id FROM animals WHERE name = 'Blossom'),
+    '2020-05-24'
+);
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES (
+    (SELECT id FROM vets WHERE name = 'William Tatcher'),
+    (SELECT id FROM animals WHERE name = 'Boarmon'),
+    '2021-01-11'
```

Comment:
Please note that Blossom visited William Tatcher on Jan 11th, 2021, not Boarmon. Kindly change this and update the query result screenshot 👍🏽